packages/jest-cli/src/reporters/CoverageReporter.js

```diff
@@ -103,11 +103,13 @@ class CoverageReporter extends BaseReporter {
       aggregatedResults.coverageMap = map;
     } catch (e) {
       console.error(
-        chalk.red(`
+        chalk.red(
+          `
```

Comment:
This is the way it was before though: https://github.com/facebook/jest/pull/3444/files#diff-cb858d9facdeae9910e3be3db8632971L106
So the fix here is gone